src/ios/OAuthPlugin.swift

```diff
@@ -37,6 +37,8 @@ class ASWebAuthenticationSessionOAuthSessionProvider : OAuthSessionProvider {
         self.aswas = ASWebAuthenticationSession(url: endpoint, callbackURLScheme: callbackURLScheme, completionHandler: { (callBack:URL?, error:Error?) in
             if let incomingUrl = callBack {
                 NotificationCenter.default.post(name: NSNotification.Name.CDVPluginHandleOpenURL, object: incomingUrl)
+            } else if let incomingUrl = error {
```

Comment:
Reusing the `incomingUrl` variable name here doesn't make the most sense, something like `else if let err = error` would be better (and then use `err` as the object passed to NotificationCenter.

For background, `error` is a `Error?` which means it might be nil.
`if let err = error` is a Swift construct that evaluates truthy if `error` is non-nil, and assigns the non-nil value to `err` (essentially changing from `Error?` to `Error`).

Reply:
Ah, thanks for the explanation!
I adapted (and retestet) the commit accordingly.